pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -49,6 +50,10 @@ import (
 	"k8s.io/kubernetes/pkg/util/tail"
 )
 
+const (
+	MaxPidsAnnotationKey = "node.alpha.kubernetes.io/max-pids"
```

Comment:
Policy is that alpha features should no longer be implemented as annotations. If we want to expose this in the API, it should be an alpha field: https://github.com/kubernetes/community/blob/master/contributors/devel/api_changes.md#alpha-field-in-existing-api-version

Reply:
ack, will try to do that